testing/cloudbuild-templates/cloud-run-anthos-template.cloudbuild.yaml

```
@@ -0,0 +1,83 @@
+steps:
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}', '$_SAMPLE']
+
+- id: 'Push Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}']
+
+- id: 'Deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  args:
+  - 'run'
+  - 'deploy'
+  - '${_SERVICE}-${COMMIT_SHA}'
+  - '--image'
+  - 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}'
+  - '--cluster'
+  - '${_CLUSTER}'
+  - '--cluster-location'
+  - '${_SERVICE_REGION}'
+  - '--platform'
+  - 'gke'
+
+- id: 'Get Cloud Run URL'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    get_url() {
+        gcloud run services describe ${_SERVICE}-${COMMIT_SHA} --format 'value(status.url)' \
+          --platform gke --cluster ${_CLUSTER} --cluster-location ${_SERVICE_REGION}
+    }
+    echo $(get_url) > _service_url
+    echo "Cloud Run URL for ${_SERVICE} is $(cat _service_url)"
+
+- id: 'Get Cloud Run External IP'
+  name: 'gcr.io/cloud-builders/kubectl'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    get_ip() {
+        gcloud container clusters get-credentials ${_CLUSTER} --zone ${_SERVICE_REGION} --project ${PROJECT_ID} \
+        && kubectl get svc istio-ingress --namespace gke-system -o yaml | sed -n -e 's/    - ip: //p'
```

Nicely done getting the this right! The Cloud Run docs are a bit inconsistent. Here's a suggestion for cleaner IP Address parsing:

```suggestion
        && kubectl get svc istio-ingress --namespace gke-system \
          --output 'jsonpath={.status.loadBalancer.ingress[0].ip}'
```